test/test-pages/table-style-attributes/expected.html

```diff
@@ -3,35 +3,29 @@
     <span size="4"><br/>...or, why do I bother.</span></b>
         </span><br/> © 2002, 2003 <a href="mailto:/dev/null@jwz.org?subject=Please%20delete%20this%20message%20without%20reading%20it.">Jamie Zawinski</a> </p>
     <div>
-        <table>
```

Comment:
This test is supposed to check that we remove attributes like `bgcolor` and `border`. Now the test no longer checks this. Instead, can you modify both `source.html` and `expected.html` to include more than 1 cell so the table stays? (maybe just split some of the paragraphs off into their own cell or something)